packages/model/README.md

```diff
@@ -0,0 +1,6 @@
+# Model
```

Comment:
I think we should follow the style most of our readmes follow. 

* Package name as the header
* A brief explanation of the package. What you have is great!
* Installation instructions
* API reference and Usage examples

The READMEs in our quilt packages are pretty well documented and follow that structure fairly well. (eg. [`@shopify/with-env`](https://github.com/Shopify/quilt/tree/master/packages/with-env), [`@shopify/async`](https://github.com/Shopify/quilt/tree/master/packages/async)). There are also a few sewing-kit-next plugins with documented READMEs ([`@sewing-kit/plugin-javascript`](https://github.com/ismail-syed/sewing-kit/tree/main/packages/plugin-javascript), [`@sewing-kit/plugin-eslint`](https://github.com/ismail-syed/sewing-kit/tree/main/packages/plugin-eslint)).